parser/svc.go

```diff
@@ -91,7 +94,58 @@ PkgLoop:
 	}
 }
 
-// parseFuncs parses the pkg for any declared RPCs and auth handlers.
+// parseServiceStruct parses the pkg for any declared encore:service structs.
+func (p *parser) parseServiceStruct(pkg *est.Package, svc *est.Service) *est.ServiceStruct {
+	var ss *est.ServiceStruct
+	for _, f := range pkg.Files {
+		for _, decl := range f.AST.Decls {
+			gd, ok := decl.(*ast.GenDecl)
+			if !ok || gd.Tok != token.TYPE {
+				continue
+			}
+
+			// We can either have the directive comment attached directly
+			// on the GenDecl, or on the TypeSpec. Handle both cases.
+			dir, doc := p.parseDirectives(gd.Doc)
+			if dir != nil && len(gd.Specs) != 1 {
+				p.err(gd.Doc.Pos(), "invalid encore directive location (expected on declaration, not group)")
```

Comment:
These aren't called `group` any more:
```suggestion
				p.err(gd.Doc.Pos(), "invalid encore directive location (expected on declaration, not a service)")
```

Reply:
This refers to `type ( /* ... */ )` which is a group of type declarations, not API groups

Reply:
Ok, I think this could be clearer in that case